okteto.yml

```diff
@@ -0,0 +1,45 @@
+name: node-compose-getting-started
+
+build:
+  api:
+    context: api
+    dockerfile: api/Dockerfile
+    image: okteto.dev/api
+
+  web:
+    context: web
+    dockerfile: web/Dockerfile
+    image: okteto.dev/web
+
+  web-dev:
+    context: web
+    target: dev
+    dockerfile: web/Dockerfile
+    image: okteto.dev/web-dev
+
+deploy:
+  compose:
```

Comment:
you can move `endpoints` here and keep docker-compose.yaml 100% compose compatible